app/views/admin/teachers/_form.html.erb

```diff
@@ -3,5 +3,7 @@
   <%= form.input :name %>
   <%= form.input :email %>
   <%= form.select :grove_id, collection: current_user.groves.map{|x| [x.name, x.id]} %>
```

Comment:
Also @Jbern16 get with @pindell-matt we found a small bug with this dropdown and his branch has the appropriate code for the collection. also, you guys might get together and abstract this form into an `admin/users/_form..` partial to dry up this code.